DISCLAIMER

```diff
@@ -0,0 +1,10 @@
+Apache Polaris (incubating) is an effort undergoing incubation at The Apache
```

Comment:
super nit: Should `Incubating` be capitalized?

Reply:
No strict requirement for caps afair